fitbenchmarking/controllers/mantid_controller.py

```diff
@@ -2,6 +2,7 @@
 Implements a controller for the Mantid fitting software.
 """
 
+from numpy.core.fromnumeric import sort
```

Comment:
@jfowkes I think I saw that this is unused. Was it used for something that was changed to function a different way?
```suggestion
```

Reply:
Yeah I saw that too, that was already there, nothing to do with me 🙂